source/gwork/include/Gwork/Controls/Property/ColorSelector.h

```diff
@@ -106,7 +106,7 @@ namespace Gwk
                 {
                     ParentClass::DoChanged();
                     unsigned int r,g,b;
-#ifdef _WIN32
+#if defined(_WIN32) && !defined(__MINGW32__)
```

Comment:
If you wanted to add platform and compiler detection and then appropriate defines so that would be very helpful. Examples I can think of a bgfx which uses [bx](https://github.com/bkaradzic/bx). And there is a big list of compiler defines on SourceForge somewhere.

Best to make a branch and submit from there once tested.